tools/functional-tester/etcd-tester/lease_stresser.go

```diff
@@ -211,13 +211,14 @@ func (ls *leaseStresser) createLeases() {
 			defer wg.Done()
 			leaseID, err := ls.createLease()
 			if err != nil {
-				plog.Errorf("lease creation error: (%v)", err)
+				plog.Debugf("lease creation error: (%v)", err)
 				return
 			}
 			plog.Debugf("lease %v created", leaseID)
 			// if attaching keys to the lease encountered an error, we don't add the lease to the aliveLeases map
 			// because invariant check on the lease will fail due to keys not found
 			if err := ls.attachKeysWithLease(leaseID); err != nil {
+				plog.Infof("unable to attach keys to lease %d error (%v)", leaseID, err)
```

Comment:
the failure can happen during failure injecting period. most of failures are expected. I'll change it to debug.